tests/test_slots.py

```diff
@@ -52,7 +53,7 @@ def test_sync_replication_slots(self):
         self.p.set_role('master')
         with mock.patch('patroni.postgresql.Postgresql.role', new_callable=PropertyMock(return_value='replica')):
             self.s.sync_replication_slots(cluster, False)
-        with patch.object(SlotsHandler, 'drop_replication_slot', Mock(return_value=True)),\
+        with patch.object(SlotsHandler, 'drop_replication_slot', Mock(return_value=(False, True))),\
```

Comment:
do we need it at all after you've added this as the default result in `MockCursor.execute()`?